designer/uix/kivy_console.py

```diff
@@ -721,10 +721,10 @@ def run_cmd(*args):
             else:
                 try:
                     command = re.sub('[ ]+', ' ', command)
-                    if command[3] == os.sep:
+                    if os.path.isdir(command[3]):
```

Comment:
Yes you are right. My fault.
At my situation command[3:] is equal to "C:\blahblah" so the first character cannot be os.sep.